src/index/updater/inscription_updater.rs

```diff
@@ -682,6 +684,21 @@ mod stream {
         .unwrap_or_else(|_| panic!("Inscription should exist: {}", self.inscription_id))
       {
         self.enrich_content(inscription);
+        let old_owner: Option<Address> = index
```

Comment:
the `&TxOut::default()` and the `None` branch are not consistent of the return I think